src/__tests__/tv.test.ts

```diff
@@ -5,21 +5,23 @@ import {tv, cn} from "../index";
 const COMMON_UNITS = ["small", "medium", "large"];
 
 const twMergeConfig = {
-  theme: {
-    opacity: ["disabled"],
-    spacing: ["divider", "unit", "unit-2", "unit-4", "unit-6"],
-    borderWidth: COMMON_UNITS,
-    borderRadius: COMMON_UNITS,
-  },
-  classGroups: {
-    shadow: [{shadow: COMMON_UNITS}],
-    "font-size": [{text: ["tiny", ...COMMON_UNITS]}],
-    "bg-image": ["bg-stripe-gradient"],
-    "min-w": [
-      {
-        "min-w": ["unit", "unit-2", "unit-4", "unit-6"],
-      },
-    ],
+  extend: {
```

Comment:
@mskelton could we handle the `extend` internally? to avoid breaking the existing twMerge configs?

Reply:
@jrgarciadev I updated the PR to support the legacy structure in addition to the new structure. We can consider the old structure deprecated and it can be removed in a future major.

Reply:
Surely we could remove the legacy settings in the next major release.